projects/templatetags/projects_extras.py

```diff
@@ -0,0 +1,80 @@
+from django import template
+from projects.models import *
+
+register = template.Library()
+
+
+@register.filter
+def get_bundles(project):
+    return BundleDetail.objects.filter(project__id=project.id).exclude(status='ARCHIVED')
+
+
+@register.filter
+def get_builds(bundle):
+    return BuildDetail.objects.filter(bundle__id=bundle.id).exclude(status='ARCHIVED')
+
+
+@register.filter
+def get_suite(bundle):
+    return SuiteDetail.objects.filter(bundle__id=bundle.id).exclude(status='ARCHIVED')[0]
```

Comment:
Ah then in that case if you're sure it'll only ever be one element, do 

```python
(element,) = my_list
``` 
This gives you several benefits:

- Python will assert that your list has exactly one element so if it doesn't, it will error rather than cause a bug
- Much easier to identify what you are doing. A `[0]` is often easy to miss.
It also works for any sort of collection and isn't exclusive to lists while `[0]` generally is